client/src/extension.ts

```diff
@@ -174,6 +175,11 @@ export async function activate (context: vscode.ExtensionContext, launchOptions:
   const workspaceFolders = _.map(_.flow(_.get('uri'), _.get('fsPath')), vscode.workspace.workspaceFolders)
   const workspaceFiles: [string] = _.map(vsCodeUriToUriString, (await vscode.workspace.findFiles(FLIX_GLOB_PATTERN)))
 
+  // Make sure we can write to `./target`
+  if (!ensureTargetWritable(_.first(workspaceFolders))) {
+    throw new Error('Cannot write to "target" folder.')
```

Comment:
should this be a notification or does an exception automatically become that?

Reply:
Uncaught exceptions are automatically picked up by VSCode. It then terminates the plugin showing the error message specified.

Reply:
Great! So this is ready for merge?

Reply:
Yes!